_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -213,8 +213,30 @@ class HealthCareApp extends React.Component {
     this.setState({ applicationData: newApplicationData });
   }
 
+  resetNullValues(objectData) {
+    const context = this;
```

Comment:
Yes... I don't really understand what's going on either. I just noticed that I didn't have access to `this` once inside the function, which confuses me because I thought lexical scoping was the advantage of arrow functions?